MediaBrowser.Providers/MediaInfo/FFProbeVideoInfo.cs

```diff
@@ -229,6 +229,12 @@ public class FFProbeVideoInfo
                 video.Video3DFormat ??= mediaInfo.Video3DFormat;
             }
 
+            if (libraryOptions.DisableEmbeddedSubtitles)
+            {
+                _logger.LogDebug("Disabling embedded subtitles due to DisableEmbeddedSubtitles setting");
```

Comment:
```suggestion
                _logger.LogDebug("Disabling embedded subtitles for {Path} due to DisableEmbeddedSubtitles setting", video.Path);
```

So we know which file the embedded subtitles are disabled for